tests/testProcessFile.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+#
+# LSST Data Management System
+# Copyright 2012-2016 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import print_function
+
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+import unittest
+
+import lsst.utils
+import lsst.utils.tests
+import lsst.pex.exceptions as pexExcept
+
+testDataPackage = "afwdata"
+try:
+    testDataDirectory = lsst.utils.getPackageDir(testDataPackage)
+except pexExcept.NotFoundError as err:
+    testDataDirectory = None
+
+
+class DoesProcessFileRun(unittest.TestCase):
```

Comment:
Generally TestCases have "Test" in their name somewhere". So, `TestProcessFileRun`.